README.md

```diff
@@ -132,7 +136,9 @@ far, `Method_1` and `Method_2` returned `'Empty` bodies, but we can fix that.
 import           Data.Text (Text)
 import qualified Serv.ContentType as Ct
 
-type Method_3 = 'Method 'GET '[] ('Body '[ Ct.TextPlain ] Text)
+type RawBody = HasBody '[ Ct.TextPlain ] Text
```

Comment:
I think a slow roll-out is valuable. We could discuss building up a single response type like `Sc.Ok ::: Respond '[] Empty` then add a header, then add a body, then combine a few possible status codes, then finally link it all together with `Method`?